src/transaction/ada.ts

```diff
@@ -0,0 +1,102 @@
+import axios from 'axios';
+import CardanoWasm from '@emurgo/cardano-serialization-lib-nodejs';
+
+import { TransferAda } from '../model';
+
+/**
+ * Prepare a signed ADA transaction with the private key locally. Nothing is broadcasted to the blockchain.
+ * @param body content of the transaction to prepare.
+ * @param graphQLUrl a Cardano GraphQL URL to get UTxOs.
+ * @returns raw transaction data in hex, to be broadcasted to blockchain.
+ */
+export const prepareADATransaction = async (body: TransferAda, graphQLUrl: string) => {
+  const fromAddress = CardanoWasm.Address.from_bech32(body.from);
+  const toAddress = CardanoWasm.Address.from_bech32(body.to);
+
+  const utxos = (
```

Comment:
this should use Tatum API, not the graphQL. Add this method as a Tatum API endpoint and obtain data from there.

Reply:
@linhdangkk Please handle this as suggested.